### PR TITLE
Transaction export dialog; wallet selection

### PR DESCRIPTION
**Pull request by yzhurko**

### CODE REVIEW

File: GUI/QtQuick/QtQuickAdapter.cpp [hunk 1]
```diff
@@ -2454,41 +2468,43 @@ void QtQuickAdapter::notifyNewTransaction(const bs::TXEntry& tx)
    }, Qt::QueuedConnection);
 }
 
+QString QtQuickAdapter::makeExportTransactionFilename(QTXSignRequest* request)
+{
+    if (request->txReq().walletIds.empty()) {
```
Comment (sergey-chernikov) [inline GUI/QtQuick/QtQuickAdapter.cpp]: wrong indentation size

File: GUI/QtQuick/QtQuickAdapter.cpp [hunk 2]
```diff
@@ -1848,6 +1857,11 @@ bs::message::ProcessingResult QtQuickAdapter::processWalletDeleted(const std::st
    pushRequest(user_, userWallets_, msg.SerializeAsString());
 
    emit successDeleteWallet();
+
+   if (walletBalances_->rowCount() > 0) {
+       emit requestWalletSelection(0);
```
Comment (sergey-chernikov) [inline GUI/QtQuick/QtQuickAdapter.cpp]: indentation

File: GUI/QtQuick/QtQuickAdapter.cpp [hunk 3]
```diff
@@ -2599,30 +2615,30 @@ bool QtQuickAdapter::isRequestReadyToSend(QTXSignRequest* request)
...
    const auto& seed = walletPropertiesModel_->seed();
    WalletBackupPdfWriter writer(
-       walletPropertiesModel_->walletId(),
-       QStringList(seed.begin(), seed.begin() + seed.length() / 2).join(QChar::fromLatin1(' ')),
-       QStringList(seed.begin() + seed.length() / 2, seed.end()).join(QChar::fromLatin1(' ')),
-       QRImageProvider().requestPixmap(walletPropertiesModel_->walletId(), nullptr, QSize(200, 200)));
+      walletPropertiesModel_->walletId(),
+      QStringList(seed.begin(), seed.begin() + seed.length() / 2).join(QChar::fromLatin1(' ')),
+      QStringList(seed.begin() + seed.length() / 2, seed.end()).join(QChar::fromLatin1(' ')),
+      QRImageProvider().requestPixmap(walletPropertiesModel_->walletId(), nullptr, QSize(200, 200)));
```
Comment (sergey-chernikov) [inline GUI/QtQuick/QtQuickAdapter.cpp:2640]: BTW, do we scale this QR code based on screen aspect ratio?

Comment (yzhurko) [reply GUI/QtQuick/QtQuickAdapter.cpp:2640]: No, it's not required to scale the size as we will draw the image into PDF.

File: GUI/QtQuick/QtQuickAdapter.cpp [hunk 4]
```diff
@@ -1848,6 +1857,11 @@ bs::message::ProcessingResult QtQuickAdapter::processWalletDeleted(const std::st
    pushRequest(user_, userWallets_, msg.SerializeAsString());
 
    emit successDeleteWallet();
+
+   if (walletBalances_->rowCount() > 0) {
+	  emit requestWalletSelection(0);
```
Comment (sergey-chernikov) [inline GUI/QtQuick/QtQuickAdapter.cpp]: indentation - it appears to be a tab + 2 spaces this time, should only be 3 spaces per level
same issue below in this file

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**